src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/Login.cshtml

```diff
@@ -22,12 +21,12 @@
 
                             <input type="hidden" asp-for="ReturnUrl" />
 
-                            <div data-test-id="email-field">
+                            <div data-test-id="field-email-address">
                                 <div data-test-id="text-field-input">
                                     <div class="nhsuk-form-group">
                                         <label asp-for="EmailAddress" class="nhsuk-label"></label>
-                                        <input asp-for="EmailAddress" class="nhsuk-input" data-test-id="login-email-address" />
-                                        <span asp-validation-for="EmailAddress"></span>
+                                        <input asp-for="EmailAddress" class="nhsuk-input" data-test-id="input-email-address" />
```

Comment:
We could include the label as part of an input tag helper but that's something we can refactor to later, if time permits.

Reply:
Yeah this is Login related, I'm moving it all to use my Input Tag Helper as part of the other user story.